src/FeedIo/Feed.php

```diff
@@ -71,7 +71,7 @@ public function current()
      */
     public function next()
     {
-        return $this->items->next();
+        $this->items->next();
```

Comment:
Are you sure for this ? I don't see why the `return` statement would be annoying